de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/HistoryStore.java

```diff
@@ -15,7 +15,8 @@
 
 	public void addHistory(IFile file, List<String> content, long timestamp) {
 		IProject project = file.getProject();
-		IFile historyFile = project.getFolder(historyFolder).getFolder(file.getProjectRelativePath().toOSString().replace(".", "_").replace("/", "_"))
+		IFile historyFile = project.getFolder(historyFolder)
+				.getFolder(file.getProjectRelativePath().toOSString().replace(".", "_").replace("/", "_"))
```

Comment:
Document purpose of this in comment.

Reply:
This problem will be addressed by the following issue #57